packages/forms-web-app/src/dynamic-forms/validator/string-validator.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-unused-vars */
+const { body } = require('express-validator');
+const BaseValidator = require('./base-validator.js');
+
+class StringValidator extends BaseValidator {
+	regex = {
+		regex: '',
+		regexMessage: ''
+	};
+	minLength = {
+		minLength: 0,
+		minLengthMessage: ''
+	};
+	maxLength = {
+		maxLength: 0,
+		maxLengthMessage: ''
+	};
+	constructor({ regex, minLength, maxLength } = {}) {
+		super();
+
+		if (!regex && !minLength && !maxLength) throw new Error('Nothing to do!');
```

Comment:
This error message could be more descriptive, while a stack trace would indicate where the error is it's helpful to have useful error messages.